digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -640,6 +650,37 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
     orderedTargets
   }
 
+  def recalculateRoadAddresses(roadNumber: Int) = {
```

Comment:
Copied code from calculator - should be changed or used directly from one class.